src/gebt_poc/gen_alpha_2D.cpp

```diff
@@ -231,25 +239,27 @@ std::tuple<State, Kokkos::View<double*>> GeneralizedAlphaTimeIntegrator::AlphaSt
         for (size_t node = 0; node < n_nodes; node++) {
             // Update the velocity, acceleration, and constraints based on the increments
             Kokkos::parallel_for(
-                size,
+                kNumberOfLieAlgebraComponents,
                 KOKKOS_LAMBDA(const size_t i) {
                     delta_gen_coords(node, i) += delta_x(i) / h;
                     velocity_next(node, i) += kGammaPrime * delta_x(i);
                     acceleration_next(node, i) += kBetaPrime * delta_x(i);
```

Comment:
Good catch, @ddement. It was a little more involved issue since `view_X.size()` returns `extent(0)` of the `Kokkos::View` - meaning `delta_x` was not correctly sized in the first place. I took care of the sizing and indexing issues for the above.